tfe/resource_tfe_workspace_test.go

```diff
@@ -311,3 +368,22 @@ resource "tfe_workspace" "foobar" {
   terraform_version = "0.11.1"
   working_directory = "terraform/test"
 }`
+
+const testAccTFEWorkspace_sshKey = `
+resource "tfe_organization" "foobar" {
+  name = "terraform-test"
+  email = "admin@company.com"
```

Comment:
I would align these assignments (run the sample tf through `terraform fmt`)

Reply:
Are we doing that for our internal tests? For docs I fully understand that, but it seems a bit much for the test fixtures of the provider?